tools/deploy-using-olm.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+
+
+# tools definitions
+K8S_CLIENT=${K8S_CLIENT:-kubectl}
+OP_SDK=${OP_SDK:-operator-sdk}
+
+# Bundle definition
+BUNDLE_REPO=${BUNDLE_REPO:-docker.io/ohadmitrani}
```

Comment:
I didn't notice this during the first look-through but you have your docker account here as the default. 
Would it be better to prompt the user for the repo if BUNDLE_REPO isn't set in the environment? 
Otherwise everyone will unintentionally try pushing to your repo, then have to go into this script to find out why.

Reply:
You are correct I will move it to be an invocation parameter instead

Reply:
@renan-campos I have pushed the changes, the script will even output an error message and terminate if no repo name  was provided via n invocation argument or BUNDLE_REPO

Reply:
Do the changes come up on your end? The commit on my side still shows 
`BUNDLE_REPO=${BUNDLE_REPO:-docker.io/ohadmitrani}`

Reply:
Should be updated now